tests/integration_tests/user_recipe_tests/test_recipe_comments.py

```diff
@@ -54,6 +54,7 @@ def test_create_comment(api_client: TestClient, unique_recipe: Recipe, unique_us
     assert response_data[0]["recipeId"] == str(unique_recipe.id)
     assert response_data[0]["text"] == create_data["text"]
     assert response_data[0]["userId"] == str(unique_user.user_id)
+    assert len(response_data[0]["user"]["fullName"]) > 0
```

Comment:
I think it would make sense to add fullName to our TestUser so we can compare the result directly. Is that something you could do?